CHANGELOG.rst

```diff
@@ -1,6 +1,106 @@
+  - [Models/Deploy][Python/R] Introduce ``pyfunc.load_model`` to be consistent with other Models modules. ``pyfunc.load_pyfunc`` will be deprecated in the near future. (#1222, @sueann)
+  - Also remove the unsupported ``jars`` argument from ```spark.log_model`` (#1222, @sueann)
+  - [Projects] Remove and rename some ``projects.run`` parameters for generality and consistency. (#1222, @sueann)
+  - [Models] Rename ``dst_path`` parameter in ``pyfunc.save_model`` to ``path`` (#1221, @aarondav)
+  - [CLI] Unify ``mlflow pyfunc`` and ``mlflow rfunc`` commands as ``mlflow models`` (#1257, @tomasatdatabricks; #1321, @dbczumar)
+  - [CLI] Turn arguments into options in ``experiments`` CLI commands (#1235, @sueann)
+  - [CLI] Consolidate ``artifacts download``, ``artifacts download-from-uri`` and ``download`` into ``artifacts download`` (#1233, @sueann)
+  - [CLI] Remove ``sagemaker list-flavors`` (#1233, @sueann)
+  - [CLI] Deprecate ``--file-store`` argument to ``mlflow server`` and ``mlflow ui`` (#1196, @smurching)
+  - [CLI] Simplify ``mlflow ui`` command by removing the ``--host`` and ``--gunicorn-opts`` options (#1267, @aarondav)
+  - [EnvVars] Prefix environment variables with "MLFLOW_" (#1268, @aarondav)
+  - [R] R flavors refactor (#1299, @kevinykuo)
+
+    - ``mlflow_predict()`` has been added in favor of ``mlflow_predict_model()`` and ``mlflow_predict_flavor()`` which have been removed.
+    - ``mlflow_save_model()`` is now a generic and ``mlflow_save_flavor()`` is no longer needed and has been removed.
+    - ``mlflow_predict()`` takes ``...`` to pass to underlying predict methods.
+    - ``mlflow_load_flavor()`` now has the signature ``function(flavor, model_path)`` and flavor authors should implement ``mlflow_load_flavor.mlflow_flavor_{FLAVORNAME}``. The flavor argument is inferred from the inputs of user-facing ``mlflow_load_model()`` and does not need to be explicitly provided by the user.
+
+  - [R] The ``mlflow_run`` API for running MLflow projects has been modified to more closely reflect the Python ``mlflow.run`` API. In particular, the order of the ``uri`` and ``entry_point`` arguments has been reversed and the ``param_list`` argument has been renamed to ``parameters``. (#1265, @smurching)
+  - [R] The ``mlflow_cli`` and ``crate`` APIs are now private. Also, the return values of experiment CRUD APIs have been updated to more closely match the REST API. In particular, ``mlflow_create_experiment`` now returns a string experiment ID instead of an experiment, and the other APIs return NULL. (#1246, @smurching)
+  - [R] Remove ``mlflow_snapshot`` and ``mlflow_restore_snapshot`` APIs. Also, the ``r_dependencies`` argument used to specify the path to a packrat r-dependencies.txt file has been removed from all APIs. (#1263, @smurching)
+
+- [Artifacts] In the Models and Projects APIs, an artifact's location is now represented as a URI. (#1190, #1254, @dbczumar; #1174, @dbczumar, @sueann; #1206, @tomasatdatabricks)
```

Comment:
This also seems like a big change that should be mentioned early on, under new features.